<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="expenses">
     <attribute name="title">
      <string>Расходы</string>
     </attribute>
     <widget class="QLCDNumber" name="expenses_sum">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>391</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>371</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Общая сумма расходов:</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="spending_date">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>411</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>240</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Метод оплаты:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="payment_method">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сумма:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="spending_sum">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>190</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="spending_description">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>270</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>290</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(при желании):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>270</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Описание</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="spending_time">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>152</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дата:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Время:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="making_expense_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>390</y>
        <width>331</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Внести расход</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="income">
     <attribute name="title">
      <string>Доходы</string>
     </attribute>
     <widget class="QLCDNumber" name="income_sum">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>391</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>361</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Общая сумма доходов:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="making_income_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>761</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Внести доход</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="income_date">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>351</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sum_of_income">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>180</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сумма:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>230</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Описание</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>250</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(при желании):</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="income_description">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дата:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="transactions">
     <attribute name="title">
      <string>Транзакции</string>
     </attribute>
     <widget class="QTableWidget" name="transactions_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>771</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_history_btn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить историю</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="exit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти из аккаунта</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
